app/templates/results.html

```diff
@@ -0,0 +1,93 @@
+<head>
+    <title> SilverScreen </title>
+
+    {# Load stylesheets #}
+    {% load static %}
+    <link rel="stylesheet" type="text/css" href="{% static "style.css" %}" />
+    <script type="text/javascript" src="{% static "js/jquery.js" %}"></script>
+    {# Load the tag library #}
+    {% load bootstrap3 %}
+
+    {# Load CSS and JavaScript #}
+    {% bootstrap_css %}
+    {% bootstrap_javascript %}
+</head>
+
+<body>
+	<nav class="navbar navbar-default">
+	  <div class="container-fluid">
+		<div class="navbar-header">
+		  <a class="navbar-brand" href="#">
+			SilverScreen
+		  </a>
+		       <form class="navbar-form navbar-left">
+				<div class="form-group">
+					<input type="text" class="form-control" placeholder="Search">
+				</div>
+				<button type="submit" class="btn btn-default">Submit</button>
+			</form>
+		</div>
+	  </div>
+	</nav>
+
+
+
+    <div class="align-left" style="width: 60%; margin: auto;">
+        <h1> {{ movie.Title }}</h1>
+        <hr>
+        <div class="well full-width clearfix">
+            {% if movie.Poster %}
+                <div class="col-sm-4">
+                    <img src="{{ movie.Poster }}" class ="center-block img-responsive" style="max-height:300px; margin-right:25px;"/>
+                </div>
+                <div class="col-sm-8">
+            {% endif %}
+                <h3 class="info-header">IMDB Rating: {{movie.imdbRating}}</h3>
+                <h3 class="info-header">Rotten Tomatoes Rating: {{movie.tomatoRating}}</h3>
+                <h3 class="info-header"> Sentiment Score:
+                    {% if overall_score <= 0 %}
+                        <strong style="color: red;">{{ overall_score }}</strong> </h3>
+                    {% else %}
+                        <strong style="color: green;">{{ overall_score }}</strong> </h3><br>
+                    {% endif %}
+                <div class="col-sm-3" style="margin-left: 0px !important;">
+                    <p>Genre: </p>
+                    <p>BoxOffice: </p>
+                    <p>Director: </p>
+                    <p>Year: </p>
+                    <p>Tomato Consensus: </p>
+                </div>
+                <div class="col-sm-9">
+                    <p>{{movie.Genres}}</p>
+                    <p>{{movie.BoxOffice}}</p>
+                    <p>{{movie.Director}}</p>
+                    <p>{{movie.Year}}</p>
+                    <em>"{{movie.tomatoConsensus}}"</em>
+                </div>
+            {% if movie.Poster %}
+                </div>
+            {% endif %}
+        </div>
+
+        {% for tweet in tweets %}
+            <hr>
+            <div class="container">
```

Comment:
This is the ugly version of this page. My pull request has the prettier, up-to-date version. We should merge it before we deploy ;)